source/ncengine/graphics2/frontend/subsystem/TransformCache.cpp

```diff
@@ -64,6 +65,7 @@ void TransformCache::UpdateMatrices(ecs::ExplicitEcs<Transform> ecs)
 
         const auto handle = static_cast<TransformDataHandle>(i);
         data[handle].modelMatrix = pool.Get(entity).TransformationMatrix();
+        data[handle].invModelMatrix = DirectX::XMMatrixTranspose(DirectX::XMMatrixInverse(nullptr, data[handle].modelMatrix));
```

Comment:
br00tal. This does result in a noticeable drop in performance. I don't think it will matter for the game though b/c we can pack all the statics up front.

Reply:
Removed